mlflow/utils/rest_utils.py

```diff
@@ -356,7 +358,18 @@ def extract_all_api_info_for_service(service, path_prefix):
     return res
 
 
-def get_single_trace_endpoint(request_id):
+def get_single_trace_endpoint(request_id, is_databricks=False):
```

Comment:
```suggestion
def get_single_trace_endpoint(request_id, use_v3=False):
```

nit: to be more honest